docs/source/main_classes/deepspeed.mdx

```diff
@@ -31,7 +31,7 @@ won't be possible on a single GPU.
 
 🤗 Transformers integrates [DeepSpeed](https://github.com/microsoft/DeepSpeed) via 2 options:
 
-1. Integration of the core DeepSpeed features via [`Trainer`]. This is everything done for your type
+1. Integration of the core DeepSpeed features via [`Trainer`]. This is everything done for you type
```

Comment:
```suggestion
1. Integration of the core DeepSpeed features via [`Trainer`]. This is an everything-done-for-you type
```
I don't understand that sentence. If "everything done for you" is to be interpreted as an adjective, it should have the hyphen and an article before, but I'm not sure if this is what you meant.

Reply:
Yes, I like your suggestion, @sgugger - thank you!